src/iotjs_binding.c

```diff
@@ -129,18 +129,18 @@ static iotjs_jval_t iotjs_jval_create_raw(jerry_value_t val) {
 }
 
 
-iotjs_jval_t* iotjs_jval_get_undefined() {
-  return &jundefined;
+iotjs_jval_t iotjs_jval_get_undefined() {
```

Comment:
Cant we remove this function? Or is it a follow-up patch? I think the change will be something similar, so I would just remove it here.

Reply:
Yes, we can. I'll update the PR, if no one has objection.